website/src/css/custom.scss

```diff
@@ -9,5 +8,7 @@
 @import 'navbar';
 @import 'showcase-link';
 @import 'showcase-section';
-@import 'utilities';
 @import 'markdown';
+
+@import url('https://fonts.googleapis.com/css2?family=Nunito+Sans:wght@300;700&display=swap');
```

Comment:
Please follow the suggestion of Asaf and move also these imports to `_fonts.scss`.

Reply:
Thanks Yarik. Moved both google fonts imports to the `_fonts.scss` file